src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -452,7 +453,7 @@ export default class AircraftController {
             altitude: spawnPatternModel.altitude,
             speed: spawnPatternModel.speed,
             heading: spawnPatternModel.heading,
-            position: spawnPatternModel.position,
+            position: spawnPatternModel.position.toPositionModel(),
```

Comment:
It's a `StaticPositionModel`, which is converted to a `PositionModel`, which is a problem as discussed because `StaticPositionModel extends PositionModel`.

But it would be illogical to have the spawn pattern's spawn position be a non-static `PositionModel`, as the spawn point itself doesn't move. It's just that we take this static position and hand it to the aircraft, who needs a dynamic position, hence the conversion.

Ideas on how else to deal with this?